0_intro/exercises/cat/src/main.rs

```diff
@@ -1,7 +1,14 @@
+use std::fs;
+
 pub fn cat(path: &str) -> String {
-    todo!();
+    fs::read_to_string(path).expect("The file doesn't exist!")
```

Comment:
good work, `expect` is usually preferred to `unwrap` as it makes finding the issue as a developer easier.